var/spack/repos/builtin/packages/hevea/package.py

```diff
@@ -28,6 +29,7 @@ class Hevea(MakefilePackage):
     # Dependency demands ocamlbuild
     depends_on("ocaml")
     depends_on("ocamlbuild")
+    conflicts("ocaml@5.0.0:", when="@:2.35")
```

Comment:
Can we express that as an additional `depends_on`:
```python
depends_on("ocaml@:xxx", when="@:2.35")
```
?

Reply:
Sure. Done.